packages/operator/pkg/testhelpers/stubclients/stub_clients.go

```diff
@@ -0,0 +1,106 @@
+package stubclients
+
+import (
+	"github.com/odahu/odahu-flow/packages/operator/pkg/apis/training"
+	"github.com/odahu/odahu-flow/packages/operator/pkg/apis/packaging"
+	"github.com/odahu/odahu-flow/packages/operator/pkg/errors"
+	"github.com/odahu/odahu-flow/packages/operator/pkg/repository/util/filter"
+)
+
+type TIStubClient struct {
+	db map[string]training.ToolchainIntegration
+}
+
+func NewTIStubClient() TIStubClient{
+	return TIStubClient{
+		db: make(map[string]training.ToolchainIntegration),
+	}
+}
+
+func (t TIStubClient) GetToolchainIntegration(name string) (*training.ToolchainIntegration, error) {
+	entity, ok := t.db[name]
+	if !ok {
+		return nil, errors.NotFoundError{Entity: name}
+	}
+	return &training.ToolchainIntegration{
+		ID:     name,
+		Spec:   entity.Spec,
+		Status: entity.Status,
+	}, nil
```

Comment:
As I understand it is the same to the following. Isn't it?
```suggestion
	return &entity, nil
```